webclient/http1/src/main/java/io/helidon/webclient/http1/Http1ClientRequestImpl.java

```diff
@@ -179,6 +180,14 @@ protected MediaContext mediaContext() {
         return super.mediaContext();
     }
 
+    @Override
+    protected void additionalHeaders() {
+        super.additionalHeaders();
+        if (proxy() != null && proxy().type() != ProxyType.NONE) {
```

Comment:
proxy() must never be null - this check should be useless.

Reply:
Done